packages/firestore/test/unit/specs/limbo_spec.test.ts

```diff
@@ -439,25 +439,27 @@ describeSpec('Limbo Documents:', [], () => {
     // This tests verifies that a document is consistent between views, even
     // if the document is only in Limbo in one of them.
     const originalQuery = Query.atPath(path('collection'));
-    const filteredQuery = Query.atPath(path('collection'))
-      .addFilter(filter('matches', '==', true));
+    const filteredQuery = Query.atPath(path('collection')).addFilter(
+      filter('matches', '==', true)
+    );
 
     const docA = doc('collection/a', 1000, { matches: true });
     const docADirty = doc(
       'collection/a',
       1000,
-      {  matches: true },
+      { matches: true },
```

Comment:
Hrm. Strange that prettier allowed an extra space before...

Reply:
It didn't. But the change was never pushed.